pyproject.toml

```diff
@@ -13,6 +13,13 @@ pydantic = "<3"
 pika = "^1.3.1"
 snowflake-sqlalchemy = ">=1.5.0"
 
+[tool.poetry.group.sql.dependencies]
+great-expectations = { version = "^0.17.19", extras = [
+    "databricks",
+    # includes redshift, postgres, etc.
+    "sqlalchemy1"
```

Comment:
This was failing before because the `sqlalchemy` gx extra requires `SQLALchemy` 2, which is not compatible with all of our datasources yet.